### PR TITLE
fix: Blank page on mobile & dapp webpage doesn't load on Safari

### PR DESCRIPTION
**Pull request by imanshu03**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

About # (link your issue here)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/components/NetworkModal/index.tsx [hunk 1]
```diff
@@ -229,7 +229,7 @@ export default function NetworkModal(): JSX.Element | null {
                                 ) {
                                     console.log(key.toString(16))
                                     if (isMetaMask) {
-                                        ;(ethereum as any).request({
+                                        ;(ethereum as any)?.request({
```
Comment (sirpy) [inline src/components/NetworkModal/index.tsx:232]: this is already behind the isMetamask guard

File: src/components/NetworkModal/index.tsx [hunk 2]
```diff
@@ -240,7 +240,7 @@ export default function NetworkModal(): JSX.Element | null {
                                     }
                                 } else {
                                     if (isMetaMask) {
-                                        ;(ethereum as any).request({
+                                        ;(ethereum as any)?.request({
```
Comment (sirpy) [inline src/components/NetworkModal/index.tsx:243]: this is already behind isMetamask guard

**Review by sirpy — CHANGES_REQUESTED** [review]
@imanshu03 have you verified that now it works on a browser without metamask?
if so you can merge after you remove redundant `?`

**Comment by imanshu03** [comment]
@sirpy Yes I have verified the application is not showing blank screen and is getting loaded on Safari on MacOS, Safari on iOS, Chrome on iOS and Chrome on Android.

**Pull request merged**